src/panels/lovelace/cards/hui-sensor-card.ts

```diff
@@ -146,7 +146,7 @@ class HuiSensorCard extends LitElement implements LovelaceCard {
   }
 
   public static getStubConfig(): object {
-    return {};
+    return { entity: "" };
```

Comment:
For a future PR, let's automatically select an entity from unused entities and fill it in the config.